src/Nordea/Components/TextInput.elm

```diff
@@ -48,6 +49,7 @@ type alias Config msg =
     , showError : Bool
     , maxLength : Maybe Int
     , pattern : Maybe String
+    , disabled : Bool
```

Comment:
Instead of adding it as an explicit property, I suggest to pass in the Html attribute directly where it is needed instead. For example:

```elm
TextInput.init ""
    |> TextInput.withPlaceholder "Text"
    |> TextInput.view [ Attr.disabled True ]
```

Reply:
Aaah 👌 Will change it to this ☺️